source/future.tex

```diff
@@ -1001,7 +1004,7 @@
  positions the output sequence	\\ \rowsep
 \tcode{(which \& (ios::in |}\br
 \tcode{ios::out)) == (ios::in |}\br
-\tcode{ios::out))} and either\br
```

Comment:
I'll try to apply this change separately ahead of time. I don't think our commit messages should discuss details of tool use. (They should only describe the change in the commit.)

Reply:
Sure; I have no strong opinions here.  I just felt silly both about making a one-character commit and about claiming authorship of something that I literally did because of the "Mismatched parentheses" message.

Reply:
I understand. I don't think commit messages are generally the right place to claim authorship, especially in this kind of document that already has a somewhat formalized notion of authorship. The PR conversation thread seems like the right place for this sort of narrative (as supporting material for how this change was motivated)... although I'd go as far as to say that obvious errors are their own, sufficient rationale for their fix.

Reply:
Of course we don't need a _rationale_ for fixing it.  I was just (very literally) surprised by the process, and wrote that down (in a way that was at least circumstantially relevant to the other changes).  Anyway, sorry for the noise.

(As far as authorship: we do have a structured field for the purpose, so maybe I should use `git commit --amend --author="Emacs <emacs-devel@gnu.org>"` instead?)

Reply:
Fixed by b2e7b06b5855bc43d92bde86c0fd716f4a847b22.

Rebase, Dr Herring.

Reply:
Done (in an appropriately boring fashion).